src/kubectlUtils.ts

```diff
@@ -112,6 +115,19 @@ export async function getDataHolders(resource: string, kubectl: Kubectl): Promis
     return depList.result.items;
 }
 
+export async function getGlobalResources(kubectl: Kubectl, resource: string): Promise<KubernetesObject[]> {
+    const rsrcs = await kubectl.asJson<KubernetesCollection<Namespace>>(`get ${resource} -o json`);

```

Comment:
Collection of `Namespace`?  Should it be `KubernetesObject`?

Reply:
Switched to `KubernetesResource`